components/contract-components/contract-release-form/index.tsx

```diff
@@ -144,21 +144,19 @@ export const ContractReleaseForm: React.FC<ContractReleaseFormProps> = ({
 
   const ensQuery = ens.useQuery(address);
 
+  const ensNameOrAddress = useMemo(() => {
+    return ensQuery?.data?.ensName || ensQuery.data?.address;
+  }, [ensQuery.data]);
+
   const successRedirectUrl = useMemo(() => {
     if (
       (!ensQuery.data?.ensName && !ensQuery.data?.address) ||
       !publishMetadata.data?.name
     ) {
       return undefined;
     }
-    return `/${ensQuery.data.ensName || ensQuery.data.address}/${
-      publishMetadata.data.name
-    }`;
-  }, [
-    ensQuery.data?.address,
-    ensQuery.data?.ensName,
-    publishMetadata.data?.name,
-  ]);
+    return `/${ensNameOrAddress}/${publishMetadata.data.name}`;
+  }, [ensQuery.data, ensNameOrAddress, publishMetadata.data?.name]);
```

Comment:
this now needs to only be ensNameOrAddress, since that's what ur using inside, just fix the `if` above and also check ensNameOrAddress there